lib/client.js

```diff
@@ -914,6 +914,98 @@ BucketsMdapiClient.prototype.versionInternal = function _version(opts, cb) {
 };
 
 
+/**
+ * Gets a batch of garbage.
+ *
+ * @param {Object} opts              - request parameters
+ * @param {String} opts.request_id   - request uuid
+ * @param {Function} cb              - callback
+ */
+BucketsMdapiClient.prototype.getGarbageBatch =
+function getGarbageBatch(opts, cb) {
```

Comment:
I don't have a particular preference for this module but, other than `ping` and `versionInternal`, the other methods on`BucketsMdapiClient` tend to just have a flat list of arguments:

```
BucketsMdapiClient.prototype.deleteObject =
    function deleteObject(owner, bucket_id, name, vnode, req_id, cb) {
```

rather than all args in a single `opts` argument.  Any particular reason?

Reply:
There's no reason I didn't follow the convention except that I'm used to how we do things elsewhere. But I'll change to be consistent here with the rest of this module. Thanks.